pkgs/servers/http/lighttpd/default.nix

```diff
@@ -1,15 +1,22 @@
 { lib, stdenv, buildPackages, fetchurl, pkg-config, pcre, libxml2, zlib, bzip2, which, file
-, openssl, enableMagnet ? false, lua5_1 ? null
+, openssl
+, enableDbi ? false, libdbi ? null
+, enableMagnet ? false, lua5_1 ? null
 , enableMysql ? false, libmysqlclient ? null
 , enableLdap ? false, openldap ? null
+, enablePam ? false, linux-pam ? null
+, enableSasl ? false, cyrus_sasl ? null
 , enableWebDAV ? false, sqlite ? null, libuuid ? null
 , enableExtendedAttrs ? false, attr ? null
 , perl
 }:
 
+assert enableDbi -> libdbi != null;
 assert enableMagnet -> lua5_1 != null;
 assert enableMysql -> libmysqlclient != null;
 assert enableLdap -> openldap != null;
+assert enablePam -> linux-pam != null;
+assert enableSasl -> cyrus_sasl != null;
 assert enableWebDAV -> sqlite != null;
 assert enableWebDAV -> libuuid != null;
 assert enableExtendedAttrs -> attr != null;
```

Comment:
This ``? null`` things can go all away. Those packages are never null by default and this only causes more issues than it fixes.

Reply:
Alright, I removed both the `null` defaults and the corresponding asserts.